CTFd/api/v1/helpers/request.py

```diff
@@ -79,6 +98,10 @@ def decorator(func):
         @wraps(func)
         def wrapper(*args, **kwargs):
             data = ARG_LOCATIONS[location]()
+
+            if not validate:
+                return func(*args, data, **kwargs)
+
```

Comment:
I also just realised that if we add these decorators to existing endpoints, we'll have to change them to access the parameter data from the function arguments instead of from the request directly. We could maybe just make it not pass the data if validate is set to false. An alternative would be to do some inspection of the function to see if it will accept the argument, but not sure if we want to go down that route. 

```suggestion
            if not validate:
                return func(*args, **kwargs)

            data = ARG_LOCATIONS[location]()

```